pkg/relayer/session/errors.go

```diff
@@ -8,4 +8,5 @@ var (
 	ErrSessionTreeNotClosed         = sdkerrors.Register(codespace, 2, "session tree not closed")
 	ErrSessionStorePathExists       = sdkerrors.Register(codespace, 3, "session store path already exists")
 	ErrSessionTreeProofPathMismatch = sdkerrors.Register(codespace, 4, "session tree proof path mismatch")
+	ErrUndefinedStoresDirectory     = sdkerrors.Register(codespace, 5, "undefined stores directory")
```

Comment:
This goes a long way when we hit real issues

```suggestion
	ErrSessionTreeUndefinedStoresDirectory      = sdkerrors.Register(codespace, 5, "directory undefined for where to key-value stores should be saved on disk")
```